ArtemShchepets/src/week1/tests/testBill.java

```diff
@@ -0,0 +1,81 @@
+package week1.tests;
+
+import org.junit.Assert;
+import org.testng.annotations.Test;
+import week1.data.Bill;
+import week1.data.Product;
+import week1.data.Seller;
+import week1.data.Time;
+
+public class testBill {
+
+    Product testProduct1 = new Product("Milk", 11.20, "#03242341");
+    Product testProduct2 = new Product("Cheese", 2.05, "#0341");
+    Product testProduct3 = new Product("Water", 33.5, "#01");
+    Product testProduct4 = new Product("Cake", 7.55, "#222");
+
+    Product[] testProductList = new Product[4];
+
+    Seller testSeller = new Seller("Valya", 22);
+    Time testTime = new Time(12, 33, 50);
+
+    Bill testBill = new Bill(testProductList, testSeller, testTime);
+
+    @Test
+    public void testAddProducts() {
+
+        testProductList[0] = testProduct1;
+        testProductList[1] = testProduct2;
+        testProductList[2] = testProduct3;
+        testProductList[3] = testProduct4;
+
+        testBill.addProduct(testProduct1);
+        testBill.addProduct(testProduct2);
+        testBill.addProduct(testProduct3);
+        testBill.addProduct(testProduct4);
+
+        Assert.assertArrayEquals(testProductList, testBill.getBillList());
+    }
+
+    @Test
+    public void testAmountPrice() {
+
+
+        double expected = testProduct1.getPrice() + testProduct2.getPrice()
+                + testProduct3.getPrice() + testProduct4.getPrice();
+
+        testBill.calculateBill();
+
+        Assert.assertEquals(expected, testBill.getBillCost(), 0);
+    }
+
+    @Test
+    public void testShowInfo() {
+
+        testBill.setClosed(false);
+
+        String expected = "***BILL***\n"
+                + testProduct1.showInfo() + "\n"
+                + testProduct2.showInfo() + "\n"
+                + testProduct3.showInfo() + "\n"
+                + testProduct4.showInfo() + "\n"
+                + "12:33:50\n" + "Seller: Valya, age: 22" + "\n***BILL IS OPENED***";
+
+        String actual = testBill.showInfo();
+
+        boolean isSame = expected.equals(actual);
+
+        Assert.assertTrue(isSame);
+    }
+
+    @Test
+    public void testCloseBill() {
```

Comment:
you may remove expected and actual variables

Reply:
https://github.com/presly808/aco22/pull/11/commits/8cc7e9b1f5cb4f1e5443a47c26e9cc20acc8e872 - fixed